app/assets/stylesheets/application.css.scss

```diff
@@ -44,7 +50,22 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 }
 
 #footer {
-  font-size: 10px;
+  height: 20px;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #080808;
+  vertical-align: middle;
+  line-height: 20px;
+  p {
+    text-align: center;
+    margin: 0;
+    font-size: 10px;
+    a{
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space